package.json

```diff
@@ -27,7 +27,9 @@
     "bootstrap": "^4.4.1",
     "chart.js": "^2.9.3",
     "codemirror": "^5.52.2",
+    "dateformat": "^4.5.1",
     "highlight.js": "^9.18.1",
+    "humanize-duration": "^3.25.1",
```

Comment:
Isn't `moment.js` enough to support our requirement?

We have this in our dependencies already.

Reply:
I didn't notice this. I'll try to migrate to it.